### PR TITLE
Remove CML usage & update docs for atlas `0.2.0`

### PR DESCRIPTION
**Pull request by sourabhxyz**

### CODE REVIEW

File: src/components/browser-integration.tsx [hunk 1]
```diff
@@ -392,17 +360,17 @@ const BrowserFunctions = () => {
         console.log(body);
```
Comment (Dacomis) [inline src/components/browser-integration.tsx:360]: Is this `console.log` needed?

File: src/components/browser-integration.tsx [hunk 2]
```diff
@@ -392,17 +360,17 @@ const BrowserFunctions = () => {
...
         const { data } = await axios.post("http://localhost:8081/betref/place", body);
         console.log(data);
```
Comment (Dacomis) [inline src/components/browser-integration.tsx:362]: Is this `console.log` needed?

File: src/pages/getting-started/browser-integration.mdx [hunk 3]
```diff
@@ -5,64 +5,48 @@ import { Callout } from 'nextra-theme-docs'
...
+
+// Create request body for calling our endpoint
+const body = {
+  arsUsedAddrs: await api.getUsedAddresses(),
+  arsChangeAddr: await api.getChangeAddress(),
+  ...(0 in colls && { arsCollateral: colls[0] }),
+  arsPutAddress: convertAddrToRaw(values.putAddress), // implementation detail
+  arsBetParams: processBrpParams(brpParams), // implementation detail
+};
+console.log(body);
```
Comment (Dacomis) [inline src/pages/getting-started/browser-integration.mdx:31]: Is this `console.log` needed?

File: src/pages/getting-started/browser-integration.mdx [hunk 4]
```diff
@@ -5,64 +5,48 @@ import { Callout } from 'nextra-theme-docs'
...
+
+// Call endpoint
+const { data } = await axios.post("http://localhost:8081/betref/add-ref-script", body);
+console.log(data);
```
Comment (Dacomis) [inline src/pages/getting-started/browser-integration.mdx:35]: Is this `console.log` needed?

File: src/components/browser-integration.tsx (hunk 1 above)
Comment (sourabhxyz) [reply src/components/browser-integration.tsx:360]: yup, it helps me knowing the contents of computed `body`.

**Review by Dacomis — APPROVED** [review]

**Comment by Dacomis** [comment]
I have tested locally the changes and everything works without any errors.
Good job, @sourabhxyz! 👏🏼

**Pull request merged**